scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/ConverterProvider.scala

```diff
@@ -210,7 +210,14 @@ private[types] object ConverterProvider {
       val name = symbol.name.toString
       val tpe = symbol.asMethod.returnType
 
-      val tree = q"$fn.get($name)"
+      
```

Comment:
Actually `tree` can be null if the type is `Option`, handled in `def option()` above. We should move this null check into the `else` branch, before `cast`.

Reply:
We need to check it in `else if` also since `REPEATED` fields aren't supposed to be null.